scripts/helpers/normalize_helpers.rb

```diff
@@ -62,60 +36,14 @@ def warn_if_already_seen(id)
   @already_seen_ids.add(id)
 end
 
-# The order of term baking/numbering differs between Kitchen and Easybake, so mask the term numbers
-
-def mask_term_numbers(element)
-  return unless element[:'data-type'] == 'term' || element[:class]&.split(' ')&.include?('os-term-section-link')
-
-  element[:id] = element[:id]&.gsub(/_term(\d+)$/, '_termXXX') if element[:id]
-  element[:href] = element[:href]&.gsub(/_term(\d+)$/, '_termXXX') if element[:href]
-end
-
-def div_to_h3_note(element)
-  unless element.parent[:'data-type'] == 'note' && \
-         element[:class]&.split(' ')&.include?('os-title') && element.name != 'h3'
-    return
-  end
-
-  element.name = 'h3'
-end
-
-def eoc_metadata_title(element)
-  unless element[:'data-type'] == 'metadata' && \
-         element.parent[:class]&.split(' ')&.include?('os-eoc')
-    return
-  end
-
-  title = element.parent.element_children.first.text
-  element.element_children.first.content = title
-end
-
-def add_index_comment(element)
-  return unless element[:class]&.split(' ')&.include?('os-term-section-link')
-
-  element.add_next_sibling('<!--
-    -->')
-end
-
 # Main normalize function for an XML document
 
-def normalize(doc, args: [])
+def normalize(doc)
   doc.traverse do |child|
-    mask_copied_id_numbers(child)
     next if child.text? || child.document?
 
-    remove_bogus_number_from_unnumbered_tables(child)
-    remove_blank_classes(child)
     sort_classes_strip_whitespace(child)
     sort_attributes(child)
```

Comment:
devs - left these in bc they appealed to my sense of order, but ... do we really need them?

Reply:
I don't think we _really_ need them. The only case I can really think of is if we change the order in which some classes/attributes were added... but even then, we'd want to know.